Orbitersdk/samples/ProjectApollo/src_saturn/sivb.cpp

```diff
@@ -1736,6 +1815,10 @@ void SIVB::CreatePayload() {
 	case PAYLOAD_ASTP:
 		plName = "ProjectApollo/ASTP";
 		break;
+	case PAYLOAD_CUSTOM:
+		if (customPayloadClass == "") { strcpy(customPayloadClass, "ProjectApollo/ASTP"); } //Avoid weird stuff in case the user doesn't load a vessel class
```

Comment:
customPayloadClass is a char array, can you even do == "" with it? And plName = customPayloadClass;?

Reply:
> And plName = customPayloadClass;?

This is the variable that it already uses to load the LM or ASTP DM vessel class.  (Line 1853)  Idk why

> customPayloadClass is a char array, can you even do == "" with it?

Looks like you can't, I'll fix that

Reply:
I meant that " plName = customPayloadClass" has a similar issue and might need to be strcpy instead. Char arrays, always causing trouble, at least to my brain.

Reply:
It didn't work right when I tried to use strcpy

Reply:
Actually no, plName is just a pointer instead of a char array. Argh, I've never properly understood these string definitions :D

Reply:
Yeah they are really confusing and annoying

Reply:
I now think plName = customPayloadClass is fine. plName will be pointing to customPayloadClass  and gets used later in the function, so it will give the right thing to oapiCreateVesselEx. I'll give it a test and then the PR should be good, the rest of the code looks fine.

Reply:
Sounds good